From dd090a2eda39631de058f046ef6c8b8419072910 Mon Sep 17 00:00:00 2001
From: graugans <k4230r6@gmail.com>
Date: Wed, 23 Jan 2013 20:00:57 +0100
Subject: [PATCH] removed the not needed subdirectory from device

I do not understand why should I use /dev/fb/0 instead of /dev/fb0
In the Kernel docs The framebuffer is documented as /dev/fb0.

http://www.kernel.org/doc/Documentation/fb/framebuffer.txt
---
 src/driver/framebuffer_ng.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/driver/framebuffer_ng.h b/src/driver/framebuffer_ng.h
index 03f0483..c10a770 100644
--- a/src/driver/framebuffer_ng.h
+++ b/src/driver/framebuffer_ng.h
@@ -166,7 +166,7 @@ class CFrameBuffer
 		void setupGXA(void);
 #endif
 
-		void init(const char * const fbDevice = "/dev/fb/0");
+		void init(const char * const fbDevice = "/dev/fb0");
 		int setMode(unsigned int xRes, unsigned int yRes, unsigned int bpp);
 
 
-- 
1.7.9.5

